tensorflow/lite/testing/generate_examples_lib.py

```diff
@@ -3690,6 +3690,12 @@ def make_sqrt_tests(options):
   return _make_elementwise_tests(tf.sqrt)(options)
 
 
+@register_make_test_function()
+def make_isfinite_tests(options):
+  """Make a set of tests to do is_finite."""
+  return _make_elementwise_tests(tf.is_finite)(options)
```

Comment:
instead of using _make_elementwise_tests, I think you need change it a little bit, since the generate test case will never populate infinite data, the default range is -100 to 100,  see https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/testing/generate_examples_lib.py#L271-L288

Reply:
@renjie-liu , thanks for pointing this out, i have updated the code as per your suggestion, which randomises the nan placement. Hope this is fine, kindly have a look.

Regards
Amit